shard-operator-parent/shard-operator-v2/src/main/java/com/redhat/service/smartevents/shard/operator/v2/ManagedBridgeService.java

```diff
@@ -21,4 +23,8 @@ public interface ManagedBridgeService {
     KnativeBroker fetchOrCreateKnativeBroker(ManagedBridge managedBridge, ConfigMap configMap);
 
     AuthorizationPolicy fetchOrCreateBridgeAuthorizationPolicy(ManagedBridge managedBridge, String path);
+
+    boolean isBridgeStatusChange(ManagedBridge updatedBridge);
```

Comment:
Minor note, this method makes the assumption that it is invoked _before_ the actual resource is updated on k8s side.
We might make it "stateless" moving the fetch outside and transform the method as `compareStatus(old, new)`
Btw it is safe in a reconcile loop so I'm fine also with this code, just a minor comment

Reply:
done.